### PR TITLE
Modify harbor-shell UI

### PR DESCRIPTION
**Pull request by AllForNothing**
1. Modify Harbor layout
image
2. UI Design: https://www.figma.com/proto/0bSkyzvchcOCrwSWROIivtIb/Harbor-v1.9?node-id=1251%3A1888&viewport=639%2C1697%2C0.5&scaling=min-zoom
3. Temporary available env. :  https://10.160.194.191
Signed-off-by: sshijun <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/app/interrogation-services/interrogation-services.component.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.sub-header-title {
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.sub-nav-bg-color {
+  background-color: #fafafa;
+}
+
+.subnav .nav {
```
Comment (jwangyangls) [inline src/portal/src/app/interrogation-services/interrogation-services.component.scss]: subnav{ .nav

File: src/portal/src/app/interrogation-services/interrogation-services.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  constructor(
+      private router: Router,
+      private route: ActivatedRoute,
+  ) {}
+
+  ngOnInit() {
+    // check scanner tab by default
+    if (!this.route.children.length) {
+      this.router.navigate(['scanners'], {relativeTo: this.route});
```
Comment (jwangyangls) [inline src/portal/src/app/interrogation-services/interrogation-services.component.ts]: ?

File: src/portal/src/app/interrogation-services/interrogation-services.component.scss (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/app/interrogation-services/interrogation-services.component.scss]: fixed

File: src/portal/src/app/interrogation-services/interrogation-services.component.ts (hunk 2 above)
Comment (AllForNothing) [reply src/portal/src/app/interrogation-services/interrogation-services.component.ts]: Removed this code.   And now use "redirect" in the routing module

**Pull request merged**